docs/README.md

```diff
@@ -1,12 +1,24 @@
 # Kremling
-Kremling is an npm library for writing css with React. It uses [React hooks](https://reactjs.org/hooks), is only 4kb gzipped,
-and requires no changes to your webpack config or build process.
+Kremling is an npm library for writing CSS with React. It is only 4kb gzipped, and requires no changes to your webpack config or build process.
 
 [Kremling Github page](https://github.com/CanopyTax/kremling)
 
 [Walkthrough](/walkthrough/getting-started.md)
 
-## Hooks example
+## Motivation
+Why use Kremling over say, CSS Modules?
+
+1. **Option to use [React hooks](https://reactjs.org/hooks)**. Kremling is bleeding edge. :)
```

Comment:
I'd rather not have hooks listed as a motivation for kremling, but a feature of kremling.

Bleeding edge is not really true since hooks are now part of react's official latest release. Additionally, I don't think that riding the bleeding edge is actually a good reason to use something. And hooks won't be new for very long, so there's a chance that this documentation becomes stale and feels out of date within a few months.

Reply:
Ok. I agree with this. I wanted to follow up with some type of comment for why Kremling worked to support hooks.